pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -708,7 +695,7 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard, ICo
         // all other token weights accordingly.
         // Clean up data structures and update the token count
         delete _tokenState[token];
-        _denormWeightSum -= _denormalizeWeight(tokenNormalizedWeight, _denormWeightSum);
+        _denormWeightSum -= tokenNormalizedWeight.mulUp(_denormWeightSum);
```

Comment:
DId you feel these didn't add much?

Reply:
Through offloading most of the usages into `ManagedPoolTokenLib` this ended up being the only place where `_denormalizeWeight` was used. I think at this point inlining it makes sense.